docusaurus-docs/community/contribute/local-setup-labextension.md

```diff
@@ -9,7 +9,7 @@ description: Local development setup for jupyterlab-conda-store
 
 - Local [clone of your fork](community/contribute/contribute-code#setup-for-local-development) of the [`jupyterlab-conda-store-ui` repository](https://github.com/conda-incubator/jupyterlab-conda-store)
 - [NodeJS `> 18`](https://nodejs.org/en/download/) installed on your local computer to build the extension package.
-- Python `>= 3.8` (and `pip`)
+- Python `>= 3.10` (and `pip`)
```

Comment:
Which makes me think... we would need to change the min supported version in the JLab extension, unless this has been done.

Reply:
100% - I've made an issue: https://github.com/conda-incubator/jupyterlab-conda-store/issues/69